internal/core/core.go

```diff
@@ -491,6 +493,59 @@ func (cs *coreService) SendMessage(ctx context.Context, options SendMessageOptio
 	return messageResponse["message_id"], nil
 }
 
+// SendMultipartMessage publishes a multipart message to a room.
+func (cs *coreService) SendMultipartMessage(
+	ctx context.Context,
+	options SendMultipartMessageOptions,
+) (uint, error) {
+	if len(options.Parts) == 0 {
+		return 0, errors.New("You must provide at least one message part")
+	}
+
+	if options.SenderID == "" {
+		return 0, errors.New("You must provide the ID of the user sending the message")
+	}
+
+	requestBody, err := common.CreateRequestBody(map[string]interface{}{"parts": options.Parts})
+	if err != nil {
+		return 0, err
+	}
+
+	response, err := common.RequestWithUserToken(
+		cs.underlyingInstance,
+		ctx,
+		options.SenderID,
+		client.RequestOptions{
+			Method: http.MethodPost,
+			Path:   fmt.Sprintf("/rooms/%s/messages", options.RoomID),
+			Body:   requestBody,
+		})
```

Comment:
Formatting seems off here. The `)` should be on the next line?

Reply:
Agreed. It's a copy paste from the method above, but I'll fix it here at least :)